ArduPlane/GCS_Mavlink.cpp

```diff
@@ -157,6 +157,35 @@ void GCS_MAVLINK_Plane::send_attitude() const
         omega.z);
 }
 
+void GCS_MAVLINK_Plane::send_attitude_target() {
+
```

Comment:
Sorry, I just remembered something. We need to define this out if quadplane is not enabled.
```suggestion
#if HAL_QUADPLANE_ENABLED
```

and a `#endif // HAL_QUADPLANE_ENABLED` at the end